### ISSUE TITLE
[Screen Reader - Makecode - Feedback Survey] Screen Reader is not reading the label information for Radio buttons and Checkboxes.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2622]
**User Experience**
If the screen reader will not give label information for Radio buttons and Checkboxes then user with visual disabilities might confuse to select the check box and radio buttons.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool: Color contrast Analyzer

**Repro Steps**
1-Navigate to url:- https://makecode.com/
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Settings control lying in the header section on the page and select it.
4-Navigate to the Give feedback menu option listed under the Settings menu control and select it.
5-Navigate to the various controls lying in the pane and verify the issue. 

**Actual Result**
Screen Reader is not reading the label information for Radio buttons and Checkboxes.

**Expected Result**
Screen Reader should read the label information for Radio buttons and Checkboxes.

**Note:**
Same issue exists after pressing enter on the Next button on the same screen.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/Documents/LCA/MAS%20-%20July%201,%202017%20Release/MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx?d=w54f28d1fa2d14dcd84e15c9b87e8aba4&csf=1

**Suggested fix**
 1. Roles associated with Labels: sections/regions of the webpage should be aligned with label’s  
2. e.g. div role="application" aria-labelledby="p123" aria-describedby="info
4. Using tag:-<h1 id="p123" Calendar 
5. Using <p id="info" 
6. This calendar shows the game schedule for the Boston Red Sox. 

**Refer Attachment**
[MAS 1.3.1_Feedback_Lablel information.zip](https://github.com/Microsoft/pxt/files/1193378/MAS.1.3.1_Feedback_Lablel.information.zip)

**Comment by veler** [comment]
This website as an external website, we don't have access to it and cannot do any change.

**Comment by v-amik** [comment]
As per process, bug can't be closed until it get assigned to HCL for regression

**Comment by microsoftsam** [comment]
The website the feedback links to is out of SCOPE for testing. @v-amik please verify and close.

**Comment by abchatra** [comment]
@microsoftsam we need to go through exception process before closing. @Jaqster is on tracking the same.

**Comment by veler** [comment]
Needs to be forwarded to [Survey Monkey](https://www.surveymonkey.com).

**Comment by Jaqster** [comment]
Submitted via their Accessibility Feedback Form: https://help.surveymonkey.com/articles/en_US/kb/508-Compliance

**Comment by abchatra** [comment]
External bug with Survey Monkey
